apps/src/applab/designMode.js

```diff
@@ -1033,4 +1116,4 @@ designMode.addKeyboardHandlers = function () {
 
 designMode.resetIds = function() {
   elementLibrary.resetIds();
-};
+};
```

Comment:
nit: please add a trailing newline, and make sure `rake lint:javascript` is passing. If you install our [commit hooks](https://github.com/code-dot-org/code-dot-org/blob/staging/CONTRIBUTING.md#code-style) then it should catch other lint errors when you try to commit.